src/jobs/deploy.yml

```diff
@@ -43,15 +43,15 @@ parameters:
     description: If you wish to pass any additional arguments to the aws deploy command
     type: string
     default: ''
-  get-deployment-group-arguments:
-    description: If you wish to pass any additional arguments to the get-deployment-group command
+  get_deployment_group_arguments:
+    description: If you wish to pass any additional arguments to the get-deployment_group command
```

Comment:
```suggestion
    description: If you wish to pass any additional arguments to the get-deployment-group command
```

Reply:
Good catch. The command needs to be in snake case as well so I just fixed it with the latest commit and changed the command to `get_deployment_group command`

Reply:
I think this refers to the AWS CLI `get-deployment-group` [command](https://docs.aws.amazon.com/cli/latest/reference/deploy/get-deployment-group.html) that we use in the shell script - not an orb command. Therefore, it should stay in the kebab casing.